app/uk/gov/hmrc/cataloguefrontend/CatalogueController.scala

```diff
@@ -237,7 +241,7 @@ class CatalogueController @Inject() (
                       repoDetails.repoType match {
                         case RepoType.Service   => renderServicePage(repoDetails.name, repoDetails, hasBranchProtectionAuth)
                         case RepoType.Library   => renderLibrary(repoDetails, hasBranchProtectionAuth)
-                        case RepoType.Prototype => renderPrototype(repoDetails, hasBranchProtectionAuth)
+                        case RepoType.Prototype => renderPrototype(repoDetails, hasBranchProtectionAuth).map(Ok(_))
```

Comment:
if we make all renderXX functions return `HTML` we can move the` map(Ok(_))` after the match